packages/dx-react-grid-material-ui/src/templates/export-button.jsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+
+export const ExportButton = ({ onClick, getMessage, ...restProps }) => (
+  <Button onClick={onClick} {...restProps}>
+    {getMessage('export')}
+  </Button>
+);
+
+ExportButton.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  getMessage: PropTypes.func.isRequired,
+};
```

Comment:
Should we provide `className` too?

Reply:
`className` prop is not used in this component, so it will be applied with `restProps`